docs/source/blocks/settings.md

```diff
@@ -172,6 +172,10 @@ initialValue({id, value, formData, intl}) => newFormData
 A must-have for modern Volto blocks, `blockSchema` is a function, or directly the schema object, that returns the schema for the block data.
 Although it's not required, defining the schema enables the block to have its initial value based on the default values declared in the schema.
 
+### `disableEnter`
+
+Normally when a block is selected and you press {kbd}`enter`, a new block is inserted below.
+When you don't want this behavior and want to handle the {kbd}`enter` input yourself inside the block you can set `disableEnter` to `true`.
```

Comment:
Something got lost in my earlier suggestion. This is grammatically correct.

```suggestion
When you don't want this behavior and want to handle the {kbd}`enter` input yourself inside the block, set `disableEnter` to `true`.
```

Reply:
Fixed